src/getMigrateFilters.ts

```diff
@@ -25,10 +25,14 @@ export const getMigrateFilters =
       errorReport,
       counters,
       doesReportIncludeStacks,
+      // `dataModels` is not used yet, but needs to be in the function's signature.
+      // eslint-disable-next-line
```

Comment:
The point is to pass it to the callback: it wouldn't be useful if we just put it in the signature.

But if you want let's just merge without the `dataModels` for now.
I have it ready on a subsequent branch, so we can add it from there.